dev-packages/e2e-tests/test-applications/remix-hydrogen/tests/server-transactions.test.ts

```diff
@@ -40,7 +40,6 @@ test('Sends two linked transactions (server & client) to Sentry', async ({ page
 
   const httpServerTraceId = httpServerTransaction.contexts?.trace?.trace_id;
   const httpServerSpanId = httpServerTransaction.contexts?.trace?.span_id;
-  const loaderSpanId = httpServerTransaction?.spans?.find(span => span.op === 'function.remix.loader')?.span_id;
```

Comment:
There was a loader span here, and not anymore. This was probably a result of unnecessary rewrapping. And that span didn't contribute to the span linking on Hydrogen.